robowflex_ompl/src/ompl_trajectory.cpp

```diff
@@ -33,3 +33,17 @@ ompl::geometric::PathGeometric OMPL::OMPLTrajectory::toOMPLPath(const ompl::geom
 
     return path;
 }
+
+void OMPL::OMPLTrajectory::fromOMPLPath(const robot_state::RobotState &reference_state,
+                                        const ompl::geometric::PathGeometric &path)
+{
+    const auto &mbss = std::dynamic_pointer_cast<ompl_interface::ModelBasedStateSpace>(
```

Comment:
Might want to throw in an error check to see if mbss is null in case a junk path is thrown in.